src/border.jl

```diff
@@ -29,30 +29,102 @@ NoPad
 Base.getindex(np::NoPad) = np.border
 
 """
-`Pad` is a type that stores choices about padding. Instances must set `style`, a
-Symbol specifying the boundary conditions of the image, one of:
-
-- `:replicate` (repeat edge values to infinity)
-- `:circular` (image edges "wrap around")
-- `:symmetric` (the image reflects relative to a position between pixels)
-- `:reflect` (the image reflects relative to the edge itself)
-
-The default value is `:replicate`.
-
-It's worth emphasizing that padding is most straightforwardly specified as a string,
-
-    imfilter(img, kernel, "replicate")
```

Comment:
I removed it because there isn't actually any method in Pad or Fill which takes a string.  It seems odd to mention it here, since it is a description of how to use the imfilter API.  I am in the process of reworking and extending the imfilter documentation in a different branch.